### PR TITLE
AG-7: Subtask - Python script that extracts all text from an url

### PR DESCRIPTION
**Pull request by lgaleana-llm**
This PR introduces a new Python script that extracts all text from a given URL. The script uses the BeautifulSoup library to parse the HTML content of the webpage and extract the text. The extracted text is then printed to the console.

### Test Plan

pytest

### CODE REVIEW

File: scripts/url_text_extractor.py [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import requests
+from bs4 import BeautifulSoup
+
+
+def extract_text_from_url(url):
+    response = requests.get(url)
+    soup = BeautifulSoup(response.text, 'html.parser')
```
Comment (lgaleana-llm) [inline scripts/url_text_extractor.py:7]: Remove empty lines, new lines, etc.

Comment (lgaleana) [inline scripts/url_text_extractor.py:6]: Remove empty lines.

Comment (lgaleana) [reply scripts/url_text_extractor.py:6]: Remove empty lines from the scraped text.

Comment (lgaleana) [reply scripts/url_text_extractor.py:6]: Make sure that text in each html tag is separated by a space.

File: scripts/url_text_extractor.py [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import requests
+from bs4 import BeautifulSoup
+
+def extract_text_from_url(url):
+    response = requests.get(url)
+    soup = BeautifulSoup(response.text, 'html.parser')
+    for br in soup.find_all("br"):
+        br.replace_with(" ")
```
Comment (lgaleana) [inline scripts/url_text_extractor.py]: No. I want to extract all text in the website. But the text from each html tag must be separated by a string space.

**Review by lgaleana — APPROVED** [review]

**Pull request merged**